src/auth-service/models/Network.js

```diff
@@ -36,6 +36,11 @@ const NetworkSchema = new Schema(
       unique: true,
     },
     net_status: { type: String, default: "inactive" },
+    net_connection_string: { type: String },
```

Comment:
@BenjaminSsempala the newly added fields for `Networks`. 
![Screenshot 2023-10-13 at 17 52 34](https://github.com/airqo-platform/AirQo-api/assets/1590213/e13edec2-a256-482d-bbca-09102deb39d7)